client/src/components/Ticketing/mainpage/hero.tsx

```diff
@@ -35,7 +54,12 @@ const Hero = () => {
               </div>
             </div>
           </div>
-          <div className='m-auto rounded-xl overflow-x-auto scroll-smooth'>
+          <button onClick={handlePrevious} className='mx-4 hidden md:inline-block'>
+            <svg xmlns="http://www.w3.org/2000/svg" className="h-10 w-10" viewBox="0 0 20 20" fill="white">
```

Comment:
For the arrow buttons, it might be a good idea to have hover effects for them. I think this would work by making `fill="currentColor"`, and then assigning text color classes to the parent `<button />`. For example `text-color-number` to make it white with a corresponding `hover:text-color-number` class to have it darken on hover. You can look at this site (https://tailwindcss.com/docs/text-color) for the color options.